### PR TITLE
Add Clusters management: HTML/REST/Model

### PR DESCRIPTION
**Pull request by Sgitario**
Fix https://github.com/halkyonio/primaza-poc/issues/31

### CODE REVIEW

**Comment by Sgitario** [comment]
I'd expect more changes to the model, for example:
- If claims or services are tied to some environments, should the environment be an entity as well?
- What is the relationship between Cluster and Service? Should we link these entities?
Anyway, I think these changes are good enough to have a base to continue working in.

Also, ideally, we should agree on how resources will look like (plural vs singular), so that all of them look the same. This is related to the discussion in https://github.com/halkyonio/primaza-poc/pull/33. So, I would like to wait for https://github.com/halkyonio/primaza-poc/pull/33 before merging, and then I'd be happy to update these changes and rebase if necessary.

**Review by cmoulliard — COMMENTED** [review]
Can you add a new box within the home.html page to access the screen `/clusters` ?

**Comment by Sgitario** [comment]
> Can you add a new box within the home.html page to access the screen `/clusters` ?

it's already there: https://github.com/halkyonio/primaza-poc/pull/39/files#diff-076b005d472dc57d2329beb7c2f1cdca7fe3a01e30aa6fed3f5e89e67f75c3a1R16

or do you mean something different?

**Comment by cmoulliard** [comment]
> or do you mean something different?

No. Perfect. Approved then

**Review by aureamunoz — APPROVED** [review]
LGTM

**Review by aureamunoz — COMMENTED** [review]
Sorry for this comment after having aprouved....What I don't like of this aproach is that we have 2 endpoints for the same resource `/claims`and `/claim`. I would like to have only one, this is why the refactoring in #26

**Comment by Sgitario** [comment]
> Sorry for this comment after having aprouved....What I don't like of this aproach is that we have 2 endpoints for the same resource `/claims`and `/claim`. I would like to have only one, this is why the refactoring in #26

No no, it's perfectly fine. Actually, I was not going to merge before agreeing with the REST approach and merging #26. I will wait for it.

**Comment by Sgitario** [comment]
I've already updated the PR to follow the same approach as done in #26.

File: servicebox-app/src/test/java/io/halkyon/ClustersEndpointTest.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+
+import io.quarkus.test.junit.QuarkusTest;
+
+@QuarkusTest
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+public class ClustersEndpointTest {
```
Comment (aureamunoz) [inline servicebox-app/src/test/java/io/halkyon/ClustersEndpointTest.java:18]: Very minor remark: I think the tests for others entities are named in singular. To be homogeneous it should be good also here to call it ClusterEndpointTest or rename others too

Comment (Sgitario) [reply servicebox-app/src/test/java/io/halkyon/ClustersEndpointTest.java:18]: I think it makes more sense to rename others to the plural, as we're covering several endpoints.

**Comment by Sgitario** [comment]
@aureamunoz I've already addressed your requested change. Would you mind having another look?

**Review by aureamunoz — APPROVED** [review]
LGTM

**Pull request merged**